phe/tests/util_test.py

```diff
@@ -83,6 +80,31 @@ def testExtendedEuclieanAlgorithm(self):
             self.assertEqual(r, s*a + t*b)
             self.assertEqual(r, math.gcd(a, b))
```

Comment:
I have modified the code (not pushed yet) to fallback to `fractions.gcd()`, which is available since 2.6 but deprecated since 3.5. The advantage of using a builtin function instead of hard-coded values is to be able to adapt tests easily while avoiding cementing an existing bug, and also improve the readability of the test code.

Reply:
:+1: 
I was mainly proposing hard-coding the results as we hard-code the test cases (and the result is not hard to know). But I agree that using the `gcd` method will make the test more readable.